SwiftGen.playground/Sources/SwiftGenStoryboardEnumBuilder.swift

```diff
@@ -4,7 +4,8 @@ import Foundation
 
 public final class SwiftGenStoryboardEnumBuilder {
     private typealias Scene = (storyboardID: String, customClass: String?)
```

Comment:
`StoryboardElement` seems ok. `Entry` could do it too I guess.
I see the idea behind `UIElement` but think it's took generic (and somehow makes me think about `AccessibilityElement`, not sure why)

Or… we could also add another `typealias`, even if it's equal to the same tuple type, just to clarify things (and to be more flexible if some day we need to add another parameter to that tuple for scenes but not segues or the other way around)?